server/src/main/java/org/apache/cloudstack/vm/UnmanagedVMsManagerImpl.java

```diff
@@ -1494,7 +1494,8 @@ protected VMTemplateVO getTemplateForImportInstance(Long templateId, Hypervisor.
         if (templateId == null) {
             template = templateDao.findByName(VM_IMPORT_DEFAULT_TEMPLATE_NAME);
             if (template == null) {
-                template = createDefaultDummyVmImportTemplate(false);
+                boolean isKVM = Hypervisor.HypervisorType.KVM == hypervisorType;
```

Comment:
Can we remove this line and pass the expression as a parameter on the line below?